projects/addon-doc/src/internal/header/header.component.ts

```diff
@@ -18,7 +18,7 @@ export class TuiDocHeaderComponent {
     private readonly stream$ = new Subject<boolean>();
 
     readonly open$ = merge(
-        this.router.events.pipe(mapTo(false)),
+        this.router.events.pipe(map(() => false)),
```

Comment:
```suggestion
        this.router.events.pipe(map(ALWAYS_FALSE_HANDLER)),
```

https://github.com/Tinkoff/taiga-ui/blob/4b799b5f5fb98eeda678b5aa7af82f9901544667/projects/cdk/constants/always-false-handler.ts#L4

Reply:
fixed